fix: author constraint missed rr.res_role extra table (#515)
astropy/pyvo: CHANGES.rst

Enhancements and Fixes
----------------------

> - Fix ``pyvo.registry.Author`` that missed a table join [#515]

Reviewer: I feel this can be rephrased a bit as it would be more informative for the users to know what it fixed rather than what the underlying bug was.
```suggestion
- Fix ``pyvo.registry.Author`` to allow registry searches with author constraints. [#515]
```
